server/poll.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+type Poll struct {
+	Creator  string
+	Question string
+	Options  []*Option
+}
+
+type Option struct {
```

Comment:
Similarly, I think that `Option` should have `(option)ID`.
Considering Poll as data, I think that it would be better if we could identify Option one by one.